moveit_core/utils/src/robot_model_test_utils.cpp

```diff
@@ -117,8 +117,11 @@ RobotModelBuilder::RobotModelBuilder(const std::string& name, const std::string&
 void RobotModelBuilder::addChain(const std::string& section, const std::string& type,
                                  const std::vector<geometry_msgs::msg::Pose>& joint_origins, urdf::Vector3 joint_axis)
 {
-  std::vector<std::string> link_names;
-  boost::split_regex(link_names, section, boost::regex("->"));
+  // https://stackoverflow.com/questions/53849/how-do-i-tokenize-a-string-in-c
+  std::regex re("->");
+  auto const link_names = std::vector<std::string>(std::sregex_token_iterator{ begin(section), end(section), re, -1 },
+                                                   std::sregex_token_iterator{});
```

Comment:
In this block of code, it seems to me that the boost version is more readable and nicer than the alternative. I'd actually prefer to keep boost.